### DISCUSSION TITLE
[@types/async] Doesn't support native async functions

### DISCUSSION THREAD

**Discussion by snoack** [DefinitelyTyped/DefinitelyTyped#68188] [Issues with a @types package]
From the [documentation](https://caolan.github.io/async/v3/global.html#AsyncFunction):
> Wherever we accept a Node-style async function, we also directly accept an [ES2017 async function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function).

However, this isn't reflected in the type definitions in the `@types/async` package. For example, following code results into a type error, while it otherwise works as expected:

```ts
  await eachLimit(items, 20, async item => {
    ...
  });
```

[a comment by an automated account is not shown]

**Comment by Juliiii** [comment]
你好！我已经收到你的邮件，我会尽快回复你的~
